codespell_lib/data/dictionary_informal.txt

```diff
@@ -1,3 +1,4 @@
+dedupe->deduplicate
```

Comment:
It's failing on this:
E           error 'dedupe' should be in aspell for dictionary /home/travis/build/codespell-project/codespell/codespell_lib/tests/../data/dictionary_informal.txt

So do we relax the typo side checks for informal, or add a informal_rare for informal words that aren't in the dictionary?

Dedupe isn't in the large dictionary even.

Reply:
Later.